bn/build.boot

```diff
@@ -0,0 +1,27 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.7.1" :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "4.11.8")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+  pom  {:project     'cljsjs/bn
+        :version     +version+
+        :description "BigNum in pure javascript"
+        :url         "https://github.com/indutny/bn.js"
+        :license     {"MIT" "https://github.com/indutny/bn.js#license"}
+        :scm         {:url "https://github.com/cljsjs/packages"}})
+
+(deftask package []
+  (comp
+    (download :url "https://wzrd.in/standalone/bn.js@4.11.8")
```

Comment:
Please use lib-version here so the url doesn't need to be updated separately:

`(download :url (format "https://wzrd.in/standalone/bn.js@%s" +lib-version+))`